### PR TITLE
Support SQLite as data store; Implement V2 API for fetching direct observations

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: sqlite/import.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```
Comment (pradh) [inline sqlite/import.sh:1]: Do you mean to checkin the datacommons.db?

File: cmd/main.go [hunk 2]
```diff
@@ -59,8 +63,13 @@ var (
 	customBigtableInfo = flag.String("custom_bigtable_info", "", "Yaml formatted text containing information for custom Bigtable")
 	// Branch Bigtable Cache
 	useBranchBigtable = flag.Bool("use_branch_bigtable", true, "Use branch bigtable cache")
+	// SQLite database
+	useSQLite = flag.Bool("use_sqlite", false, "Use SQLite as database.")
+	sqliteDB  = flag.String("sqlite_db", "./sqlite/datacommons.db", "SQLite DB file path.")
 	// Stat-var search cache
 	useSearch = flag.Bool("use_search", true, "Uses stat var search. Will build search indexes.")
+	// Include maps client
+	useMapsApi = flag.Bool("use_maps_api", true, "Uses maps API for place recognition.")
```
Comment (pradh) [inline cmd/main.go:72]: Is this for the recon API implementation?

File: internal/server/v2/observation/direct.go [hunk 3]
```diff
@@ -48,70 +49,123 @@ func FetchDirect(
...
-							FacetId:      facetID,
-							Observations: obsList,
-						},
-					)
 				}
+				result.ByVariable[variable].ByEntity[entity] = entityObservation
 			}
-			result.ByVariable[variable].ByEntity[entity] = entityObservation
+		}
+	} else if store.SQLiteClient != nil {
```
Comment (pradh) [inline internal/server/v2/observation/direct.go:125]: nice.  Maybe could have a TODO to modularize the different backends a bit?

**Review by pradh — APPROVED** [review]
Very cool!

File: cmd/main.go (hunk 2 above)
Comment (shifucun) [reply cmd/main.go:72]: Yes.

File: internal/server/v2/observation/direct.go (hunk 3 above)
Comment (shifucun) [reply internal/server/v2/observation/direct.go:125]: Yea, that sounds good. I will do that after adding more APIs / need a new connector for Cloud SQL

File: sqlite/import.sh (hunk 1 above)
Comment (shifucun) [reply sqlite/import.sh:1]: Yea, I want to expand that and use it for testing; Also users would use it as "hello world" db.

**Review by shifucun — COMMENTED** [review]
thanks!

**Pull request merged**